packages/server/lib/modes/run.js

```diff
@@ -1077,28 +1080,22 @@ module.exports = {
       return this.currentSetScreenshotMetadata(data)
     }
 
-    const wait = () => {
+    const wait = async () => {
```

Comment:
Shouldn't really matter here, but any reason to add the `async` if we're already returning a Promise? Adds an extra wrapping Promise, though it's likely a negligible cost unless this is used frequently.

Reply:
Fixed - https://github.com/cypress-io/cypress/pull/17200/commits/53d3a62efac2921d8d3fe2b33bae141ac52ba741

Had to wrap `navigateToNextSpec` in `Promise.resolve` since it no longer returns a promise (not that returning a promise ever made sense - all this function does is ultimately emit an event anyway, nothing to wait for).